webpack.config.babel.js

```diff
@@ -83,13 +85,26 @@ module.exports = createConfig([
               return module.resource && module.resource.indexOf(path.resolve(__dirname, 'src')) === -1
             },
             name: 'vundle',
-            chunks: 'all'
-          }
-        }
+            chunks: 'all',
+          },
+        },
       },
-    }
+    },
   }),
 
+
+  env('production', [
+    addPlugins([
+      new SentryWebpackPlugin({
+        include: '.',
+        ignoreFile: '.sentrycliignore',
+        ignore: ['node_modules', 'webpack.config.babel.js', 'presets'],
+        configFile: 'sentry.properties',
```

Comment:
1. it seems that you already have `.sentrycliignore` file, so you don't need `ignore` prop here
2. there no `sentry.properties` file in this PR